spec/controllers/identify_controller_spec.rb

```diff
@@ -370,9 +370,9 @@
               valid: false,
               error: 'self-signed cert',
               openssl_valid: false,
-              openssl_errors: 'error 18 at 0 depth lookup: self signed certificate, error 26 at 0 depth lookup: unsupported certificate purpose',
```

Comment:
This test is for self-signed certificates, so I'm surprised that it needs to have the self-signed part of the error message removed.

Reply:
yea me too. Gonna check on this more